.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+    runs-on: ubuntu-latest
+    
+    defaults:
+      run:
+        working-directory: ./src
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [8.2, 8.3]
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }} - ${{ matrix.dependency-version }}
```

actionlint (1.7.9)

property "laravel" is not defined in object type {php: number}

(expression)